src/components/PopupWithDeleteConfirm.js

```diff
@@ -0,0 +1,24 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDeleteConfirm extends Popup {
+  constructor(popupSelector, handleDeleteSubmit) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._handleCardDelete = handleDeleteSubmit;
```

Comment:
Needs correction: please remove all the card mentions from here, so the class will be general